packages/notifications/graphql/resolvers/Device.js

```diff
@@ -0,0 +1,17 @@
+const { Roles } = require('@orbiting/backend-modules-auth')
+const userAccessRoles = ['admin', 'supporter']
+const { transformUser } = require('@orbiting/backend-modules-auth')
+
+module.exports = {
+  async user (device, args, { pgdb, user: me }) {
+    const user = await pgdb.public.users.findOne({
+      id: device.userId
+    })
+      .then(user => transformUser(user))
+    Roles.ensureUserIsMeOrInRoles(user, me, userAccessRoles)
+    return user
+  },
+  lastSeen (device) {
```

Comment:
Naming: Not sure if `updatedAt` really represents a last seen timestamp.

I see `updatedAt` updated when device is added or a (device) token is rolled.

I'd feel fine with `lastSeen` if timestamp is updated everything a request from a device comes in (it does not update, does it?).

Reply:
> (it does not update, does it?)

no it doesn't on every request. But the app is supposed to call `upsertDevice` everytime it comes back from the background. So it's still fairly up to date. Do you think that's ok?

Reply:
I think that is very much so ok.